src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -72,7 +74,7 @@ public static void LogRequest(this ILogger logger, HttpRequest request, HttpResp
             PathString resourcePath = request.Path;
             string host = $"{request.Scheme}://{request.Host}";
 
-            logger.LogInformation(RequestFormat, request.Method, host, resourcePath, statusCode, duration, DateTimeOffset.UtcNow, context);
+            logger.LogInformation(RequestFormat, request.Method, host, resourcePath, statusCode, duration, DateTimeOffset.UtcNow.ToString(UniversalSortableDateTimePattern), context);
```

Comment:
When using directly `DateTimeOffset.UtcNow`, an exception was thrown in the conversion back to a `DateTimeOffset` instance.
This pattern is available in full .NET and is universal... but was wondering if there's a better approach?

Reply:
That's odd because that worked for me yesterday? 🤔 Or you mean during testing then?

Reply:
Yes, when testing. It fails when using the combination logger extensions + application insights converters.

Reply:
I'm not sure if we should change this because of testing though. What does the raw message look like without the current change?

Reply:
It shows the date in some format that the `DateTimeOffset.Parse` can't parse correctly.
So exception is thrown when you try to convert the log events to the telemetry instances.